abb_irb120_moveit_config/config/joint_limits.yaml

```diff
@@ -1,31 +1,34 @@
+# joint_limits.yaml allows the dynamics properties specified in the URDF to be overwritten or augmented as needed
+# Specific joint properties can be changed with the keys [max_position, min_position, max_velocity, max_acceleration]
+# Joint limits can be turned off with [has_velocity_limits, has_acceleration_limits]
 joint_limits:
   joint_1:
     has_velocity_limits: true
     max_velocity: 4.36332
-    has_acceleration_limits: true
-    max_acceleration: 0.87266
+    has_acceleration_limits: false
+    max_acceleration: 0
```

Comment:
This could be reverted as well.

Although those limits are made up, just as those for the 120t.